packages/lit-dev-content/site/_includes/docs-nav-collapsing.html

```diff
@@ -18,12 +18,18 @@
           <span class="sectionHead
                        {% if section.url == page.url %}active{% endif %}">
             {{ section.title }}
+            {% if section.labs == true %}
+              <img class="labs" src="/images/alerts/labs.svg" alt="labs" />
+            {% endif %}
           </span>
         </summary>
         <ol>
           {%- for child in section.children %}
             <li{% if child.url == page.url %} class="active"{% endif %}>
               <a href="{{ child.url | url }}">{{ child.title }}</a>
+              {% if child.labs == true %}
+                <img class="labs" src="/images/alerts/labs.svg" alt="labs" />
```

Comment:
Since they're both low on the page, please add [`loading="lazy"`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/loading) and [`fetchpriority="low"`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/fetchPriority) to both `<img>` elements in this file and in `docs-nav.html`